src/main/java/com/functions/Function.java

```diff
@@ -103,4 +105,29 @@ public static HttpResponseMessage HttpTriggerJavaVersion(
     public static String getJavaVersion() {
         return String.join(" - ", System.getProperty("java.home"), System.getProperty("java.version"));
     }
+
+    /**
+     * This function listens at endpoint "/api/StaticWebPage".
+     * It can be used to read and serve a static web page.
+     * Note: Read the file from the right location for local machine and azure portal usage.
+     */
+    @FunctionName("StaticWebPage")
+    public HttpResponseMessage getStaticWebPage(
+        @HttpTrigger(
+            name = "req",
+            methods = {HttpMethod.GET, HttpMethod.POST},
+            authLevel = AuthorizationLevel.ANONYMOUS)
+        HttpRequestMessage<Optional<String>> request,
+        final ExecutionContext context) {
+        context.getLogger().info("Java HTTP trigger processed a request.");
+
+        File htmlFile = new File("index.html");
```

Comment:
do we need to add a sample index.html file here? It's location is same as this file right.

Reply:
I see, I think this should be fine then.